appspec.yml

```diff
@@ -94,6 +94,9 @@ hooks:
     - location: deploy/hooks/afterinstall/compilemessages.sh
       timeout: 300
       runas: polling_stations
+    - location: deploy/hooks/afterinstall/createcachetable.sh
```

Comment:
I've actually not tested that this works in an AWS environment. Chances are we're going to need some IAM permissions, but I thought it was easier to let you do this @GeoWill, rather than me try a deploy at this stage.